tester/tests/coco_eval

```diff
@@ -0,0 +1,70 @@
+
+echo 'Run cmds with absolute path'
+/bin/ls
+
+export COCOSHELL=cocoshell
```

Comment:
I think it would be nice to have some explanations on these tests as well

Reply:
I can do it while completing it though, no worries

Reply:
oki =) 
But yeah I didn't put the explanations because I didn't have time (and probably I was too tired haha. It was like 7:30 or something). I agree with all comments.